### PR TITLE
Changes to support Boxer V2.5

### PR DESCRIPTION
**Pull request by jhiggins-cpr**

### CODE REVIEW

File: docs/robots/indoor_robots/boxer/integration_boxer.mdx [hunk 1]
```diff
@@ -55,7 +55,7 @@ Contact [Support](#support) if you need more information.
 Ensure that the Boxer is shut off and the circuit breaker is turned to the _off_ position before performing any electrical work on the Boxer.
 
 The Boxer is equipped with an attachment interface that consists of a Power-over-Ethernet (PoE) RJ45 bulkhead connector, a USB Type-A bulkhead connector, and a 37-position panel receptacle and mated plug.
-To connect with the Boxer attachment interface see Section 10 of the [OTTO 100 V2.4 Operation and Maintenance Manual](https://help.ottomotors.com/docs/files/53683025/99909709/1/1606232510624/OMM-000094-Operation+and+Maintenance+Manual+OTTO+100+V2.4_A.pdf).
+To connect with the Boxer attachment interface see Section 10 of the [OTTO 100 V2.5 Operation and Maintenance Manual](https://docs.ottomotors.com/PDFs/100-v2.5-OMM.pdf).
```
Comment (rhysfaultless-cpr) [inline docs/robots/indoor_robots/boxer/integration_boxer.mdx:58]: Do all Clearpath Customers get access to OTTO's support centre?

File: docs/robots/indoor_robots/boxer/user_manual_boxer.mdx [hunk 2]
```diff
@@ -37,28 +48,53 @@ If you elected to purchase standard payload modules or custom integration servic
...
-Below is an overview of the Boxer robot hardware that outlines the key components that you should be aware of in order to get started with using your robot.
-
-<center>
-  <img
-    src="/img/robot_images/boxer_images/robot_boxer_3.png"
-    width="800"
-  />
-</center>
+Below is an overview of the Boxer robot hardware that outlines the key components that you should be aware of in order to get started
+with using your robot. Note that the front e-stop button is only present on Boxer V2.5.
```
Comment (rhysfaultless-cpr) [inline docs/robots/indoor_robots/boxer/user_manual_boxer.mdx]: 'emergency stop' per OTTO's documentation glossary:
https://wiki.clearpathrobotics.com/pages/viewpage.action?pageId=106572126

File: docs/robots/indoor_robots/boxer/user_manual_boxer.mdx [hunk 3]
```diff
@@ -67,9 +103,9 @@ The HMI has 5 elements:
 
 1.  Ethernet port
 2.  Power button
-3.  Motion Stop reset button
+3.  E-Stop reset button
```
Comment (rhysfaultless-cpr) [inline docs/robots/indoor_robots/boxer/user_manual_boxer.mdx]: 'emergency stop'

**Review by rhysfaultless-cpr — APPROVED** [review]
Looks good. My main request is to chose a consistent style for 'emergency stop' with / without capital letters.

Note: the Boxer / OTTO 100 has a hardware dual channel Emergency Stop loop, unlike some legacy Clearpath robots.

**Review by tonybaltovski — APPROVED** [review]

File: docs/robots/indoor_robots/boxer/integration_boxer.mdx (hunk 1 above)
Comment (jhiggins-cpr) [reply docs/robots/indoor_robots/boxer/integration_boxer.mdx:58]: All Boxer customers are required to sign the EULA and have access to OTTO's support centre. That said, I think all the links, with the exception of TalentLMS, don't require user authentication.

File: docs/robots/indoor_robots/boxer/user_manual_boxer.mdx (hunk 2 above)
Comment (jhiggins-cpr) [reply docs/robots/indoor_robots/boxer/user_manual_boxer.mdx]: Updated throughout.

File: docs/robots/indoor_robots/boxer/user_manual_boxer.mdx (hunk 3 above)
Comment (jhiggins-cpr) [reply docs/robots/indoor_robots/boxer/user_manual_boxer.mdx]: Updated

**Pull request merged**